dev/docker-compose.yaml

```diff
@@ -71,6 +75,8 @@ services:
       - /frontend/dist
     command: /bin/sh -c "while true; do sleep 600; done"
     ports:
+      # Dev image with `node start` uses port 4200
+      # For the upstream dockerfile with nginx, use 4200:80
```

Comment:
maybe something like: `For the upstream dockerfile with nginx, remove the target and use 4200:80`. I think simply changing the port will not work

Reply:
For me the target bit was obvious but I kept forgetting to change the port.

Should I remove this comment altogether?

Reply:
I would say yes